.github/workflows/redpanda-build.yml

```diff
@@ -27,8 +27,11 @@ jobs:
             apt-get install -y git sudo pkg-config
 
         - name: install dependencies
-          run: ./install-dependencies.sh
-
+          run: |
+            ./install-dependencies.sh
+            apt-get install -y gcc-10 g++-10
+            update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10
+            update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 10
```

Comment:
Might be best to connect these two:
`update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10 --slave /usr/bin/g++ g++ /usr/bin/g++-10`